packages/cardano-services-client/src/version.ts

```diff
@@ -2,5 +2,5 @@
 // auto-generated using /home/rhys/code/iohk/cardano-js-sdk/packages/cardano-services-client/scripts/createVersionSource.js
 export const version = {
   api: '1.0.0',
-  software: '0.7.2'
+  software: '0.9.8'
```

Comment:
Is this file suppose to be part of the submitted files? I saw it was gitignored :thinking:

Reply:
Good catch, I'll purge it.

Reply:
Decided to keep it in source control, so removed from the ignore file.